presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergDistributedHiveUnsafe.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.hive;
+
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+@Test
+public class TestIcebergDistributedHiveUnsafe
```

Comment:
```suggestion
public class TestIcebergDistributedHiveLockDisabled
```

Also add a comment explaining the property why this needs separate tests.